src/main/webapp/script.js

```diff
@@ -12,3 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function onIndexLoad() {
+  checkLogin();
+}
+
+function checkLogin() {
+  fetch('/login').then(response => response.json()).then(status => {
+    if(status.loggedIn){
+      restrictedTabs = document.getElementsByClassName('restricted-tab');
+      for(let element of restrictedTabs) {
+        element.className = 'restricted-tab nav-item nav-link';
+      }
+      document.getElementById('login-tab').innerText = 'Logout';
+      document.getElementById('login-tab').href = status.changeLogInStatusURL;
+    } else {
+      document.getElementById('loggedOut').style.display = 'inline';
+      document.getElementById('login-message').innerHTML = 'To get started, please <a href="' + status.changeLogInStatusURL + '">login</a>.';
+      restrictedTabs = document.getElementsByClassName('restricted-tab');
+      for(let element of restrictedTabs) {
+        element.className = 'restricted-tab nav-item nav-link disabled';
+      }
+      document.getElementById('login-tab').innerText = 'Login';
+      document.getElementById('login-tab').href = status.changeLogInStatusURL;
```

Comment:
Pull these lines out of the if/else logic since it applies to both.

Reply:
Done